src/libraries/System.IO.FileSystem/tests/File/ReadWriteAllText.cs

```diff
@@ -10,11 +11,18 @@ public class File_ReadWriteAllText : FileSystemTest
     {
         #region Utilities
 
+        protected virtual bool IsAppend {  get; }
+
         protected virtual void Write(string path, string content)
         {
             File.WriteAllText(path, content);
         }
 
+        protected virtual void Write(string path, string content, Encoding encoding)
+        {
+            File.WriteAllText(path, content, encoding);
```

Comment:
```suggestion
            if (IsAppend)
            {
                File.AppendAllText(path, content, encoding);
            }
            else
            {
                File.AppendAllText(path, content, encoding);
            }   
```
Is any class calling these virtual implementations? 
It would be good to make this `throw InvalidOperationException();` so future tests don't call this expecting to honor the `IsAppend` property.